src/VscPreviewExtension/src/extension.ts

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+"use strict";
+
+import { workspace, window, ExtensionContext, commands, Event, Uri, ViewColumn, TextDocument } from "vscode";
+import * as path from "path";
```

Comment:
import * as module_name ,will there be confilct when I use 'module_name.function'?

---
In reply to: [75793806](https://github.com/dotnet/docfx/pull/616#discussion_r75793806) [](ancestors = 75793806)